src/pipelines/epidemiology/gb_authority.py

```diff
@@ -91,7 +91,7 @@ def _fix_bad_total_deceased(data) -> None:
     # Some areas have bad reporting e.g. UKC on 09/09/2020 has N/A and 0 as
     # new_deceased and total_deceased, creating an erroneous entry in the data.
     filter_query = (data.new_deceased.isna()) & (data.total_deceased == 0)
-    data.loc[filter_query, "total_deceased"] = "NaN"
+    data.loc[filter_query, "total_deceased"] = float("NaN")
```

Comment:
You can use `None` instead, which is preferred since it will go through the fast path in `lib.cast.isna`

Reply:
Done.